public/style.css

```diff
@@ -433,11 +435,58 @@ input[type=checkbox]:checked + label ~ .talent-list {
   }
 }
 
+@media only screen and (width: 667px) {
```

Comment:
do you mean max-width?

Reply:
In this case I wanted to explicitly specify the exact width; this is for when the iPhone is changed from portrait mode to landscape mode, so the triangles and a few other things had to be tweaked. I'd like to make the entire CSS more responsive and done in flex rather than using these exact patches, so I'll try to look into that.